library/core/src/mem/maybe_uninit.rs

```diff
@@ -135,10 +131,10 @@ use crate::slice;
 ///
 ///     // Everything is initialized. Transmute the array to the
 ///     // initialized type.
-///     unsafe { mem::transmute::<_, [Vec<u32>; 1000]>(data) }
+///     unsafe { MaybeUninit::<[Vec<_>; 1000]>::assume_init(data.into()) }
 /// };
 ///
-/// assert_eq!(&data[0], &[42]);
+/// assert_eq!(&data[888], &[42]);
```

Comment:
```suggestion
/// assert_eq!(&data[42], &[42]);
```
seems like the obvious choice ;)